apps/challenges/urls.py

```diff
@@ -21,5 +21,7 @@
         name='get_all_challenges'),
     url(r'challenge/(?P<pk>[0-9]+)/',
         views.get_challenge_by_pk, name='get_challenge_by_pk'),
+    url(r'challenge$', views.get_challenges_based_on_teams,
```

Comment:
afterwards we can remove challenge in one commit if `challenges/challenge` is redundant.